content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md

```diff
@@ -0,0 +1,153 @@
+page][community] which includes links to our Slack channel and community meetings.
+
+[ingc]:https://kubernetes.io/docs/concepts/services-networking/ingress/#ingress-class
+[strc]:https://kubernetes.io/docs/concepts/storage/storage-classes/
+[crd]:https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/
+[concepts]:https://gateway-api.sigs.k8s.io/concepts/api-overview/
+[guides]:https://gateway-api.sigs.k8s.io/v1alpha2/guides/getting-started/
+[impl]:https://gateway-api.sigs.k8s.io/implementations
+[install-crds]:https://gateway-api.sigs.k8s.io/v1alpha2/guides/getting-started/#install-the-crds
+[issue]:https://github.com/kubernetes-sigs/gateway-api/issues/new/choose
+[disc]:https://github.com/kubernetes-sigs/gateway-api/discussions/new
+[community]:https://gateway-api.sigs.k8s.io/contributing/community/
+
+## Release highlights
+
+### Graduation to beta
+
+The `v0.5.0` release is particularly historic because it marks the growth in
+maturity to a `v1beta1` release for some of our key APIs:
+
+- `GatewayClass`
+- `Gateway`
+- `HTTPRoute`
+
+This achievement was marked by the completion of several graduation criteria:
+
+- API has been [widely implemented][impl].
+- Conformance tests provide basic coverage for all resources and have multiple implementations passing tests.
+- Most of the API surface is actively being used.
+- Kubernetes SIG Network API reviewers have approved graduation to beta.
+
+To see what's in store for future releases check out the next steps section.
+
+[impl]:https://gateway-api.sigs.k8s.io/implementations
+
+### Release channels
+
+This release introduces the `experimental` and `standard` [release channels][ch]
+which enable a better balance of maintaining stability while still enabling
+experimentation and iterative development.
+
+The `standard` release channel includes:
+
+- resources that have graduated to beta
+- fields that have graduated to standard (no longer considered experimental)
+
+The `experimental` release channel includes everything in the `standard` release
+channel, plus:
+
+- `alpha` API resources
+- fields that are considered experimental and have not graduated to `standard` channel
+
+Release channels are used internally to enable iterative development with
+quick turnaround, and externally to indicate feature stability to implementors
+and end-users.
+
+For this release we've added the following experimental features:
+
+- [Routes can attach to Gateways by specifying port numbers](https://gateway-api.sigs.k8s.io/geps/gep-957/)
+- [URL rewrites and path redirects](https://gateway-api.sigs.k8s.io/geps/gep-726/)
+
+[ch]:https://gateway-api.sigs.k8s.io/concepts/versioning/#release-channels-eg-experimental-standard
+
+## Other improvements
+
+For an exhaustive list of changes included in the `v0.5.0` release, please see
+the [v0.5.0 release page][releasepage].
+
+[releasepage]:https://github.com/kubernetes-sigs/gateway-api/releases/tag/v0.4.3
+
+## Next steps
```

Comment:
```suggestion
## Gateway API for service mesh 

Some service mesh projects have [already implemented support for the Gateway API](https://gateway-api.sigs.k8s.io/implementations/). Significant overlap between the Service Mesh Interface (SMI) APIs and the Gateway API has [inspired discussion in the SMI community](https://github.com/servicemeshinterface/smi-spec/issues/249) about possible integration.

We are pleased to announce that the service mesh community, including representatives from Cilium Service Mesh, Consul, Istio, Kuma, Linkerd, NGINX Service Mesh and Open Service Mesh, is coming together in a new group for [Gateway API Mesh Management and Administration](https://docs.google.com/document/d/1Qt3N1RLnRom7jKNhcnNLg_xaPn0TwKGuxxgXLOguNvE/edit#).

This group will deliver [enhancement proposals](https://gateway-api.sigs.k8s.io/v1beta1/contributing/gep/) consisting of resources, additions, and modifications to the Gateway API specification for mesh and mesh-adjacent use-cases.

This work has begun with [an exploration of using Gateway API for east/west traffic](https://docs.google.com/document/d/1T_DtMQoq2tccLAtJTpo3c0ohjm25vRS35MsestSL9QU/edit#heading=h.jt37re3yi6k5) and will continue with enhancement in areas such as authentication and authorization policy. 

## Next Steps
```

Reply:
@keithmattix @howardjohn @mikemorris @robscott PTAL

Reply:
done

Reply:
LGTM. If we want GAMMA to be an acronym we may want to expand it, but I don't mind either way

Reply:
We do, later in the sentence!  Can edit around that if it's non-obvious; I tried to say explicitly it was a workstream but the sentence was super long and contained "Gateway API" three times

Reply:
Maybe bold the first letter of the expanded name for clarity?

Reply:
done

Reply:
Added in #34968